README.md

```diff
@@ -45,8 +54,9 @@ enforce the code style and linting requirements of this project.
 
 The recommended way to set this project up is using python 3.9 (or higher) with
 a standard `venv` setup.
-The project uses a "src-dir" layout, so you will need to set up `PYTHONPATH`
-with at least the `src/` directory for Python to locate the modules.
+The project uses a "src-dir" layout; for Python to be able to locate the modules, 
+you will need to set up the `PYTHONPATH` environment variable with at least the `src/` directory.
+
 For convenience in POSIX-like systems, an importable script `./tools/path`
 is provided that will return the appropriate path.
 
```

Comment:
Suspect all the docs are going to need some attention - but that can wait until this PR is on mainline